web/src/App.tsx

```diff
@@ -2,7 +2,7 @@ import { AuthProvider } from 'react-oauth2-code-pkce'
 import { authConfig } from './authConfig'
 import { MainPage } from './pages/Main'
 
-const AUTH_DISABLED = process.env.REACT_APP_AUTH !== '1'
+export const AUTH_DISABLED = process.env.REACT_APP_AUTH !== '1'
```

Comment:
I'm not sure if App.tsx is the best owner of this constant, now that it is used globally. Maybe we could store constants in a separate file, so we'll know where to find them?

Reply:
Fixed